frontends/bmd/src/pages/card_error_screen.tsx

```diff
@@ -0,0 +1,32 @@
+import React, { useEffect } from 'react';
+
+import { Main, MainChild } from '@votingworks/ui';
+
+import { Prose } from '../components/prose';
+import { Screen } from '../components/screen';
+import { triggerAudioFocus } from '../utils/trigger_audio_focus';
+
+interface Props {
+  useEffectToggleLargeDisplay: () => void;
+}
+
+export function CardErrorScreen({
+  useEffectToggleLargeDisplay,
+}: Props): JSX.Element {
+  // eslint-disable-next-line react-hooks/exhaustive-deps
+  useEffect(useEffectToggleLargeDisplay, []);
+  useEffect(triggerAudioFocus, []);
+
+  return (
+    <Screen white>
+      <Main>
+        <MainChild center>
+          <Prose textCenter id="audiofocus">
+            <h1>Card is backwards</h1>
+            <p>Remove the card, turn it around, and insert it again.</p>
+          </Prose>
```

Comment:
I followed the screen pattern of bmd here, which was different from precinct-scanner. I also didn't include a graphic since the graphics didn't seem to be set up in bmd like they are in precinct-scanner, and the other bmd error screens don't have graphics.